### ISSUE TITLE
A330-200 autopilot panel issues

### ISSUE THREAD

**Issue by hobblob** [FGMEMBERS/A330-200#10]
I am currently unable to select managed mode for altitude and V/S on the autopilot panel in the airbus a330-200. I am only able to get the plane to enter my desired altitudes. (P.S sometimes V/S seems to go into managed mode). All help is appreciated.

Cheers
hobblob

**Comment by legoboyvdlp** [comment]
You do know about pull/push ing the knob? Pull is selected; push is managed.

**Comment by FGDATA** [comment]
@legoboyvdlp, can you explain me how do you pull or push the knob. I also have the same issue of not beeing clear on how to operate those knobs,
@IAHM-COL

**Comment by legoboyvdlp** [comment]
It's simple!
Drag = rotate
Shift + Click: push
Shift + Alt + Click: pull

**Comment by IAHM-COL** [comment]
Now that's clear/ Thanks Mate

**Comment by hobblob** [comment]
This is what I am doing- I insert the altitudes into the FMC and push the knob however this only makes the plane go into level change mode and the autopilot takes the plane to the height designated on the panel. I do not have this issue on the a330-300 or the other airbus planes. P.S pulling the knob just seems to stop the plane from achieving said altitude. The main screen does display, when the knob is pushed, op descent or thr clb. Could someone explain what these mean and whether they are related to the issue?

thanks for any help given
hobblob

**Comment by legoboyvdlp** [comment]
Push will respect the constraints. Say you must be at 5000 at SABER. It will level you off whenever you reach 5000 till you pass SABER. It is CLB / DES mode on the PFD. It will set a specific thrust and use pitch for speed.
Pull will simply descend you or climb you. It won't respect the constraints. It will set a specific thrust, and then climb at the speed set in the speed window, using pitch. It is OP CLB / OP DES.

**Comment by hobblob** [comment]
I understand what you are saying but the plane wont go to any altitude set in the FMC. It will only go to the altitude set on the panel. Also whenever anything on the autopilot panel is placed in managed mode, all values are replaced by dashes showing it is in managed mode. However, this is not the case with Altitude. 

Thanks for the advice so far.

**Comment by legoboyvdlp** [comment]
1. It is supposed to do that. It is not a Boeing, remember!
2. It should?

**Comment by hobblob** [comment]
I think there is some confusion @legoboyvdlp . I remember from omega's tutorials, that on the FMC you could implement altitudes for VNAV to follow- should I not expect this? He also seems to be able to get his plane into managed mode on the autopilot panel. Maybe you could email me your procedures or what I should expect from the FMC and autopilot and what I shouldn't expect. Email: hobblob@gmail.com. 

Apologies for the confusion.  Thanks for all the help so far.
hobblob

**Comment by legoboyvdlp** [comment]
Nope, that is most definitely not correct behaviour; Airbus does not have VNAV.
Managed mode will simply descend you to the altitude, and if you must be at x altitude at y waypoint, wil llevel you as soon as you reach the altitude, then descend you as soon as you pass y waypoint.

As far as I know, the altitude window just displays a ball beside it to indicate it is managed, and nothing for selected.

**Comment by hobblob** [comment]
![capture- a330-200](https://cloud.githubusercontent.com/assets/14257626/12173367/52920616-b54e-11e5-8e87-8c32502154dc.PNG)

That is what my autopilot panel looks like when I put everything into managed mode. This is from a flight I just started from EGLL-EGCC. The plane just flew straight to 35000 instead of the pre-assigned altitudes in the FMC. It did though follow all the speeds I had entered into the FMC. You can also see that the plane is in Level change mode. I think you are not understanding my point that the autopilot won't follow the FMC which is annoying as I have to set the altitude for each waypoint manually as I come to it.

**Comment by legoboyvdlp** [comment]
That's managed, correct.
The plane is doing everything it is designed to.
It is not supposed to follow the FMC altitude. They are constraints. It will do what you tell it to on the FCU. And if you set it to climb or descend, it will try and follow those constraints.

**Comment by hobblob** [comment]
Ok. I understand now. Thank you for clearing the confusion. Would you be able to tell me the range within it will follow the constraints?

**Comment by legoboyvdlp** [comment]
If you start to descend, you will see an arrow on the screen, which I believe is blue and ___> shape, which will show where you will level off. If it is before the waypoint, you will reach the altitude before that waypoint, and can afford to descend at a slower rate, by sleecting a lower selected speed. If it is after the waypoint, the constraint symbol will be orange, and you need to extend speedbrakes.
Enjoy!

**Comment by hobblob** [comment]
Thank you so much. Everything is clear now.

**Comment by hobblob** [comment]
![capture-a330-300](https://cloud.githubusercontent.com/assets/14257626/12184107/2bbc3a2e-b58b-11e5-9ba1-2b91efb0b014.PNG)

I thought I should mention this. As you can see from the panel above all the dials are in managed mode and no values can be set on any of them (unlike the a330-200 where altitude can be inserted). This is on the a330-300. I recently carried out a flight on it; to my amazement it flew to every altitude set on the FMC without any input from me (other than the obvious turning on of the autopilot). Is this an expected difference between the a330-200 and the -300?

**Comment by legoboyvdlp** [comment]
It's just that the A330-300 is not accurately done. This is the artix A330; the other one is the Omega A330.

**Comment by hobblob** [comment]
Ok

**Comment by FGDATA** [comment]
@legoboyvdlp @hobblob 
Can you test whether the last commit reverting a merge fixes this?
59a22737e419783523554fe9123191a7eec721ce

Thanks
@IAHM-COL
